src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -68,6 +68,8 @@ def detect_sdk_crash(self, event: Event) -> Optional[Event]:
                 sdk_crash_event_data, settings.SDK_CRASH_DETECTION_PROJECT_ID
             )
 
+        return None
```

Comment:
nit: you can skip this explicit return.

Reply:
But when I remove it `make backend-typing` complains:
```
--> Running Python typing checks
mypy --strict --warn-unreachable --config-file mypy.ini
src/sentry/utils/sdk_crashes/sdk_crash_detection.py:37: error: Missing return statement  [return]
Found 1 error in 1 file (checked 798 source files)
```